multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/SystemParameters.java

```diff
@@ -133,6 +137,9 @@ private void putRoutingParameters(Module module, Map<String, Object> moduleParam
         }
         moduleSystemParameters.put(SupportedParameters.DEFAULT_URI, appendRoutePathIfPresent(defaultUri, moduleParameters));
 
+        String liveUri = getDefaultHost(module.getName()) + DOMAIN_PLACEHOLDER_SUFFIX;
+        moduleSystemParameters.put(SupportedParameters.DEFAULT_LIVE_URI, appendRoutePathIfPresent(liveUri, moduleParameters));
```

Comment:
DEFAULT_LIVE_APP_NAME is set as a reference, but this one is set as a resolved app name - could this lead to some corner cases (in case app name is later changed)? Or is this intentional to avoid idle app name?